lib/src/HalstackContext.tsx

```diff
@@ -105,10 +105,10 @@ const parseTheme = (theme) => {
 
   const dateTokens = componentTokensCopy.dateInput;
   dateTokens.pickerSelectedDateBackgroundColor =
-    theme?.dateInput?.selectedDateBackgroundColor ?? dateTokens.pickerSelectedDateBackgroundColor;
```

Comment:
`pickerSelectedDateBackgroundColor` is not existing anymore since it has been removed.